src/modules/dashboard/components/ActionsPage/FinalizeMotionWidget/VoteResults/VoteResultsItem.tsx

```diff
@@ -20,14 +20,14 @@ interface Props {
   value: number;
   maxValue: number;
   maxPercentage?: number;
-  title: MessageDescriptor;
+  title: string | MessageDescriptor;
   voters?: Address[];
   maxAvatars?: number;
 }
 
-const displayName = 'VoteResultsWidget';
+const displayName = `dashboard.ActionPage.FinalizeVoteWidget.VoteResults.VoteResultsItem`;
```

Comment:
Same here, should be `FinalizeMotionWidget`

Reply:
Good catch!